yt_dlp/extractor/tver.py

```diff
@@ -100,3 +103,47 @@ def _real_extract(self, url):
                 self.BRIGHTCOVE_URL_TEMPLATE % (p_id, r_id), {'geo_countries': ['JP']}),
             'ie_key': 'BrightcoveNew',
         }
+
+
+class TVerSeriesIE(TVerBaseIE):
+    _VALID_URL = r'https?://(?:www\.)?tver\.jp/series/(?P<id>[a-zA-Z0-9]+)'
+    _TESTS = [{
+        'url': 'https://tver.jp/series/srtxft431v',
+        'only_matching': True,
+    }]
+
+    def _real_extract(self, url):
+        playlist = []
+        series_id = self._match_id(url)
+        season_json = self._download_json(
```

Comment:
```suggestion
        series_id = self._match_id(url)
        return self.playlist_result(self._entries(series_id))

    def _entries(self, series_id):
        season_json = self._download_json(
```
and then remove an unused variable